docs/Getting Started/Fedora/Root on ZFS/6-recovery.rst

```diff
@@ -202,21 +187,21 @@ it is relatively easy to perform a system backup and migration.
 
 #. Create a snapshot of root file system::
 
-    zfs snapshot -r rpool/fedora@backup
-    zfs snapshot -r bpool/fedora@backup
+    zfs snapshot -r rpool/arch@backup
```

Comment:
Maybe fedora? Or use same `INST_ID` var everywhere.

Same to changes in this file.

Reply:
Oops, sorry for the slip-up. Fixed. Note that `arch-chroot` is a standalone program.